concepticondata/conceptlists/Lee-2015-246.tsv

```diff
@@ -0,0 +1,247 @@
+ID	NUMBER	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS
+Lee-2015-246-1	1	above	1741	ABOVE
+Lee-2015-246-2	2	all	98	ALL
+Lee-2015-246-3	3	and	1577	AND
+Lee-2015-246-4	4	animal	619	ANIMAL
+Lee-2015-246-5	5	ash	646	ASH
+Lee-2015-246-6	6	at	1461	AT
+Lee-2015-246-7	7	back	1291	BACK
+Lee-2015-246-8	8	bad, evil	3764	BAD OR EVIL
+Lee-2015-246-9	9	bark (of a tree)	1204	BARK
+Lee-2015-246-10	10	because	1157	BECAUSE
+Lee-2015-246-11	11	belly	1251	BELLY
+Lee-2015-246-12	12	below	1485	BELOW OR UNDER
+Lee-2015-246-13	13	big	1202	BIG
+Lee-2015-246-14	14	bird	937	BIRD
+Lee-2015-246-15	15	black	163	BLACK
+Lee-2015-246-16	16	blood	946	BLOOD
+Lee-2015-246-17	17	bone	1394	BONE
+Lee-2015-246-18	18	branch	1531	BRANCH
+Lee-2015-246-19	19	breast	1402	BREAST
+Lee-2015-246-20	20	child	2099	CHILD
+Lee-2015-246-21	21	claw	72	CLAW
+Lee-2015-246-22	22	cloud	1489	CLOUD
+Lee-2015-246-23	23	cold	1287	COLD
+Lee-2015-246-24	24	correct, true	1725	CORRECT (RIGHT)
+Lee-2015-246-25	25	day	1225	DAY (NOT NIGHT)
+Lee-2015-246-26	26	dirty	1230	DIRTY
+Lee-2015-246-27	27	dog	2009	DOG
+Lee-2015-246-28	28	dry	1398	DRY
+Lee-2015-246-29	29	dull	379	BLUNT
+Lee-2015-246-30	30	dust	2	DUST
+Lee-2015-246-31	31	ear	1247	EAR
+Lee-2015-246-32	32	earth/soil	1228	EARTH (SOIL)
+Lee-2015-246-33	33	egg	744	EGG
+Lee-2015-246-34	34	Eight	1705	EIGHT
+Lee-2015-246-35	35	eye	1248	EYE
+Lee-2015-246-36	36	far	1406	FAR
+Lee-2015-246-37	37	fat/grease	323	FAT (ORGANIC SUBSTANCE)
+Lee-2015-246-38	38	father	1217	FATHER
+Lee-2015-246-39	39	feather	1201	FEATHER
+Lee-2015-246-40	40	few	1242	FEW
+Lee-2015-246-41	41	Fifty	1717	FIFTY
+Lee-2015-246-42	42	fire	221	FIRE
+Lee-2015-246-43	43	fish	227	FISH
+Lee-2015-246-44	44	Five	493	FIVE
+Lee-2015-246-45	45	flower	239	FLOWER
+Lee-2015-246-46	46	fog	249	FOG
+Lee-2015-246-47	47	foot	1301	FOOT
+Lee-2015-246-48	48	Four	1500	FOUR
+Lee-2015-246-49	49	fruit	1507	FRUIT
+Lee-2015-246-50	50	full	1429	FULL
+Lee-2015-246-51	51	good	1035	GOOD
+Lee-2015-246-52	52	grass	606	GRASS
+Lee-2015-246-53	53	green	1425	GREEN
+Lee-2015-246-54	54	hair	1040	HAIR
+Lee-2015-246-55	55	hand	1277	HAND
+Lee-2015-246-56	56	he/she	2642	HE OR SHE
+Lee-2015-246-57	57	head	1256	HEAD
+Lee-2015-246-58	58	heart	1223	HEART
+Lee-2015-246-59	59	heavy	1210	HEAVY
+Lee-2015-246-60	60	here	136	HERE
+Lee-2015-246-61	61	horn	1393	HORN (ANATOMY)
+Lee-2015-246-62	62	house	1252	HOUSE
+Lee-2015-246-63	63	how?	1239	HOW
+Lee-2015-246-64	64	husband	1200	HUSBAND
+Lee-2015-246-65	65	I	1209	I
+Lee-2015-246-66	66	ice	617	ICE
+Lee-2015-246-67	67	if	1459	IF
+Lee-2015-246-68	68	in, inside	1606	INSIDE
+Lee-2015-246-69	69	intestines	1991	INTESTINES
+Lee-2015-246-70	70	knee	1371	KNEE
+Lee-2015-246-71	71	lake	624	LAKE
+Lee-2015-246-72	72	leaf	628	LEAF
+Lee-2015-246-73	73	left	244	LEFT
+Lee-2015-246-74	74	leg	1297	LEG
+Lee-2015-246-75	75	lightening	1654	LIGHTNING
```

Comment:
Is the typo in the original source as well?

Reply:
yep.